src/index.tsx

```diff
@@ -23,6 +23,13 @@ type ChartData = {
   end?: Date;
 };
 
+// Event contains label and value of current datapoint
```

Comment:
I think this is an obvious comment given the contents on the type/interface below. Let's remove this.

Reply:
Removed